purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1959,11 +1959,19 @@ class Purchases internal constructor(
          * Enable debug logging. Useful for debugging issues with the lovely team @RevenueCat
          */
         @JvmStatic
+        @Deprecated(message = "Use logLevel instead")
         var debugLogsEnabled
-            get() = Config.debugLogsEnabled
-            set(value) {
-                Config.debugLogsEnabled = value
-            }
+            get() = logLevel.debugLogsEnabled
+            set(value) { logLevel = LogLevel.debugLogsEnabled(value) }
+
+        /**
+         * Configure log level. Useful for debugging issues with the lovely team @RevenueCat
+         * By default, LogLevel.DEBUG in debug builds, and LogLevel.INFO in release builds.
+         */
+        @JvmStatic
+        var logLevel: LogLevel
+            get() = Config.logLevel
+            set(value) { Config.logLevel = value }
```

Comment:
Weird, this _is_ called by tests.

Reply:
I believe in the tests we are setting the `Config.logLevel` directly instead of going through `Purchases.logLevel` right? 

In this case, I think it might be nice to change the tests to use these methods to make sure we cover the full flow users would use.

Reply:
OMG you're right!! Thank you!
I'm gonna make it explicit instead of relying on that static import.

Reply:
This code coverage thing is useful haha

Reply:
Now I know not to assume it was wrong 🫢